python/hail/docs/hail2_tutorials/hail2-overview.ipynb

```diff
@@ -0,0 +1,1227 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Rare variant analysis\n",
+    "\n",
+    "Here we'll demonstrate how one can use the expression language to group and count by any arbitrary properties in row and column fields. Hail also implements the sequence kernel association test (SKAT).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "entries = ds.entries_table()\n",
+    "results = (entries.group_by(pop = entries.SuperPopulation, chromosome = entries.v.contig)\n",
+    "      .aggregate(n_het = agg.count_where(entries.GT.is_het())))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results.show() "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "What if we want to group by minor allele frequency bin and hair color, and calculate the mean GQ?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "entries = entries.annotate(maf_bin = functions.cond(entries.info.AF[0]<0.01, \"< 1%\", \n",
+    "                              functions.cond(entries.info.AF[0]<0.05, \"1%-5%\", \">5%\")))\n",
+    "\n",
+    "results2 = (entries.group_by(af_bin = entries.maf_bin, purple_hair = entries.PurpleHair)\n",
+    "      .aggregate(mean_gq = agg.stats(entries.GQ).mean, mean_dp = agg.stats(entries.DP).mean))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results2.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We've shown that it's easy to aggregate by a couple of arbitrary statistics. This specific examples may not provide especially useful pieces of information, but this same pattern can be used to detect effects of rare variation:\n",
+    "\n",
+    " - Count the number of heterozygous genotypes per gene by functional category (synonymous, missense, or loss-of-function) to estimate per-gene functional constraint\n",
+    " - Count the number of singleton loss-of-function mutations per gene in cases and controls to detect genes involved in disease"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Eplilogue\n",
+    "\n",
+    "Congrats! If you've made it this far, you're perfectly primed to read the [Overview](https://hail.is/hail/overview.html), look through the [Hail objects](https://hail.is/hail/types.html) representing many core concepts in genetics, and check out the many Hail functions defined in the [Python API](https://hail.is/hail/api.html). If you use Hail for your own science, we'd love to hear from you on [Gitter chat](https://gitter.im/hail-is/hail) or the [discussion forum](http://discuss.hail.is).\n",
```

Comment:
These point to hail1 docs mostly though (e.g. Hail Objects)... Is this just in anticipation of the hail2 ones taking their place?